__tests__/app.test.js

```diff
@@ -168,6 +142,59 @@ describe("GET /api/articles", ()=> {
           })
       })
   })
+  test("should sort by query", () => {
```

Comment:
even though these tests are nested in the describe block of the closing endpoint, they could still be more specific. 

- What should sort by query
- What are the possible queries?